lib/proxy/log.rb

```diff
@@ -12,79 +9,49 @@ def logger
   end
 
   class LoggerFactory
-    class LogFormatter < ::Logger::Formatter
-      Format = "%s, [%s%s] %5s -- %s: %s\n".freeze
-
-      def call(severity, time, progname, msg)
-        Format % [severity[0..0], format_datetime(time), request_id[0..7], severity, progname, msg2str(msg)]
-      end
-
-      def request_id
-        Thread.current.thread_variable_get(:request_id).to_s
-      end
-    end
-
     begin
       require 'syslog/logger'
-      ::Syslog::Logger.class_eval { alias_method :write, :info }
-
-      class SyslogFormatter < Syslog::Logger::Formatter
-        Format = "<%s> %s\n".freeze
-
-        def call(severity, time, progname, msg)
-          request_id.empty? ? clean(msg) : Format % [request_id[0..7], clean(msg)]
-        end
-
-        def request_id
-          Thread.current.thread_variable_get(:request_id).to_s
-        end
-      end
-
       @syslog_available = true
     rescue LoadError
-      # ignore, syslog isn't available on this platform
       @syslog_available = false
     end
 
     def self.logger
+      logger_name = 'foreman-proxy'
```

Comment:
shouldn't it be a configuration, and that's just the default name?

Reply:
why would you change this from configuration?

Reply:
to have the ability to set up the log system you are using in a manner that easier for you to find things out. 

For example "proxy_#xxx" so you can figure out from what proxy it arrived if you have more then one, and send it to tools such as graylog2 or other logging systems, rather then looking inside the machine at the log.

Reply:
Both rsyslog and journad provides `hostname` field for that. Can you give another example? This is just a process name actually, it should be set to application name according to `logging` documentation, it sends this into syslog or system journal as well. This should not change.